apps/microtvm/arduino/template_project/src/host_driven/platform.c

```diff
@@ -21,27 +21,32 @@
 #include "standalone_crt/include/tvm/runtime/crt/error_codes.h"
 #include "stdarg.h"
 
+// Called by TVM when a message needs to be formatted.
+__attribute__((weak)) size_t TVMPlatformFormatMessage(char* out_buf, size_t out_buf_size_bytes,
+                                                      const char* fmt, va_list args) {
+  return vsnprintf(out_buf, out_buf_size_bytes, fmt, args);
+}
+
+// Called by TVM when an internal invariant is violated, and execution cannot continue.
 // Blink code for debugging purposes
```

Comment:
Remove comment since it seems like this function does not blink.

Reply:
done